### PR TITLE
refine `executeQuery`: remove `bool internal`

### PR DESCRIPTION
**Pull request by SeaRise**
### What problem does this PR solve?

Issue Number: ref #5900

Problem Summary:

### What is changed and how it works?

ref https://github.com/pingcap/tiflash/pull/6643#discussion_r1071803855

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by ywqzzy — APPROVED** [review]

**Review by Lloyd-Pottiger — APPROVED** [review]
Others LGTM

File: dbms/src/Flash/Coprocessor/DAGDriver.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 PingCAP, Ltd.
+// Copyright 2023 PingCAP, Ltd.
```
[the remark this answers is not in the document]
Comment (SeaRise) [reply dbms/src/Flash/Coprocessor/DAGDriver.cpp:1]: no, is all files.
A pr is needed later to update the file headers of all files.

**Comment by SeaRise** [comment]
Incompatible with `DBGInvoker`

**Pull request closed without merging**